lib/smart_answer_flows/check-uk-visa/outcomes/outcome_work_y.erb

```diff
@@ -17,27 +17,27 @@
 
   A ‘skilled worker’ visa may be suitable if you’ve been offered a:
 
-  - [skilled job](https://www.gov.uk/tier-2-general) in the UK
-  - [job in public sector health or social care](https://www.gov.uk/tier-2-health-care-visa), for example for the NHS
-  - [role in your overseas employer’s UK branch](https://www.gov.uk/tier-2-intracompany-transfer-worker-visa) - the visa you apply for depends on your circumstances
-  - [job in a religious community](https://www.gov.uk/tier-2-minister-of-religion-visa)
-  - job as an [elite sportsperson or coach](https://www.gov.uk/tier-2-sportsperson-worker-visa)
+  - [skilled job](https://www.gov.uk/skilled-worker-visa) in the UK
+  - [job in public sector health or social care](https://www.gov.uk/health-care-worker-visa), for example for the NHS
+  - [role in your overseas employer’s UK branch](https://www.gov.uk/intracompany-transfer-worker-visa) - the visa you apply for depends on your circumstances
+  - [job in a religious community](https://www.gov.uk/minister-of-religion-visa)
+  - job as an [elite sportsperson or coach](https://www.gov.uk/sportsperson-visa)
 
   ###Temporary workers
 
   <% if calculator.passport_country_in_youth_mobility_scheme_list? %>
-    You can apply for the [‘youth mobility scheme’ visa](/tier-5-youth-mobility) if you want to live and work in the UK for up to 2 years.
+    You can apply for the [‘youth mobility scheme’ visa](/youth-mobility) if you want to live and work in the UK for up to 2 years.
   <% end %>
 
   A ‘temporary worker’ visa may be suitable if you want to work in the UK for a short time:
 
-  - in [sports](https://www.gov.uk/tier-5-temporary-worker-creative-and-sporting-visa)
-  - in [arts or entertainment](https://www.gov.uk/tier-5-temporary-worker-creative-and-sporting-visa)
-  - as a [volunteer for a charity](https://www.gov.uk/tier-5-temporary-worker-charity-worker-visa)
-  - in a [work experience or research and training](https://www.gov.uk/tier-5-government-authorised-exchange) programme
-  - for a [religious organisation](https://www.gov.uk/tier-5-religious)
+  - in [sports](https://www.gov.uk/temporary-worker-creative-and-sporting-visa)
+  - in [arts or entertainment](https://www.gov.uk/temporary-worker-creative-and-sporting-visa)
+  - as a [volunteer for a charity](https://www.gov.uk/temporary-worker-charity-worker-visa)
+  - in a [work experience or research and training](https://www.gov.uk/government-authorised-exchange) programme
+  - for a [religious organisation](https://www.gov.uk/religious-worker-visa)
```

Comment:
Maybe not for now, but we should probably update these to not have `gov.uk` hardcoded so they work as expected in our integration/staging environments.

Reply:
I think in the past we've used absolute URLs, so we can share a heroku link with departments to fact check - (because we can't do draft for smart answers that are already live).

Reply:
Ah good point... looks like our usage of absolute/relative links are quite inconsistent in this app anyway...